# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../luaexpat/makefile.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- luaexpat-1.0.1-vanilla/makefile	2005-06-06 22:25:23.000000000 +0200
+++ luaexpat-1.0.1/makefile	2006-03-13 18:59:48.000000000 +0100
@@ -1,17 +1,11 @@
 T= lxp
 V= 1.0.1
-CONFIG= ./config
-
-include $(CONFIG)
 
 
 lib: src/$(LIBNAME)
 
-src/$(LIBNAME) : src/lxplib.o $(COMPAT_DIR)/compat-5.1.o
-	$(CC) -o src/$(LIBNAME) $(LIB_OPTION) src/lxplib.o $(COMPAT_DIR)/compat-5.1.o
-
-$(COMPAT_DIR)/compat-5.1.o: $(COMPAT_DIR)/compat-5.1.c
-	$(CC) -c $(CFLAGS) -o $@ $(COMPAT_DIR)/compat-5.1.c
+src/$(LIBNAME) : src/lxplib.o
+	$(CC) -o src/$(LIBNAME) $(LIB_OPTION) src/lxplib.o
 
 install:
 	mkdir -p $(LUA_LIBDIR)
@@ -21,6 +15,6 @@
 	cp src/$T/lom.lua $(LUA_DIR)/$T
 
 clean:
-	rm -f src/$(LIBNAME) src/lxplib.o $(COMPAT_DIR)/compat-5.1.o
+	rm -f src/$(LIBNAME) src/lxplib.o
 
 # $Id: makefile,v 1.28 2005/06/06 20:25:23 tomas Exp $
